src/fileattributefetch.cpp

```diff
@@ -70,6 +71,7 @@ void FileAttributeFetchChannel::dispatch(MegaClient* client)
 
     if (req.outbuf.size())
     {
+        e = API_EFAILED;
```

Comment:
After this, only the transfer can fail without any API error code. I set the error to `API_EFAILED` for consistency with errors generated in regular transfers (they use `API_EFAILED` too)